code/style.css

```diff
@@ -1,4 +1,446 @@
-html {
-  background: #0025ff;
+/****** Media Query Mobile default > 650px ********/
+@import url('https://fonts.googleapis.com/css2?family=Maven+Pro:wght@500&display=swap');
+body{
+  margin: 0;
+  font-family: 'Maven Pro', sans-serif;
+}
+/*******DEFAULT SETTING*****/
+button{
+  padding: 10px; 
+  border:0.16em solid #002e63;
+  background:#002e63;
+  color:white;
+  text-align:center;
+  transition: all 0.20s; /*makes the a smooth "alower" effect on hover*/
+    }
+
+nav button:hover{
+  color:#DDDDDD;
+  border-color:#DDDDDD;
+  }
+
+
+  button:hover{
+  color:#DDDDDD;
+  border-color:#DDDDDD;
+    }
+
+a{
+  text-decoration: none;
+  color:black;
+}
+
+
+/****** NAV BAR********/
+nav{
+  display: flex; 
+  padding: 10px 15px;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: solid 0.16em;
+  margin: 2px 3%;
+}
+
+.hamburger div {
+  width: 35px;
+  height: 5px;
+  background-color: #333;
+  margin: 6px 0;
+  }
+
+nav a{
+  padding: 10px; 
+  text-decoration: none;
+  color: #312c2c;
+}
+
+.right-nav {
+  display: none; 
+}
+
+.logo{
+  height: 60px;
+  justify-content: center;
+  margin: 0;
+}
+
+/********************* BIG NEWS************/
+
+
+.big-news{
+display: flex;
+flex-direction: column-reverse;
+align-items: flex-start;
+justify-content: center;
+margin: 5px;
+height: 500px;
+margin: 3%;
+
+}
+
+
+.big-news img{
+  display: flex;
+  overflow: hidden;
+  width: 100%;
+  height: 100%;
+
+}
```

Comment:
Take a look at the properties object-fit and object-position to get rid of the distortion!